README.md

```diff
@@ -173,7 +173,29 @@ installations requiring long-term consistency.
 | [valid-expect-in-promise](docs/rules/valid-expect-in-promise.md)             | Enforce having return statement when testing with promises      | ![recommended][] |              |
 | [valid-title](docs/rules/valid-title.md)                                     | Enforce valid titles                                            | ![recommended][] | ![fixable][] |
 
-<!-- end rules list -->
+<!-- end base rules list -->
+
+## TypeScript Rules
```

Comment:
something about needing to install the upstream plugin as well? Possibly in the table if we'll have othe rrules in the future that do _not_ use type info

Reply:
>  Possibly in the table if we'll have othe rrules in the future that do not use type info

You mean if we have other rules that don't extend base rules.

-------

You're completely right - I don't expect us to make any more rules that extend base rules, only more rules that use type information; so I might leave the table as is for now and just add a sentence saying that `unbound-method` is an extension rule (as well as updating its own docs).